src/utils/ucx/ucx_utils.cpp

```diff
@@ -67,6 +67,43 @@ void ucx_modify_config(ucp_config_t *config, std::string_view key,
     }
 }
 
+[[nodiscard]] std::string to_string(ucp_err_handling_mode_t t) noexcept
```

Comment:
too general name

Reply:
IMO very bad style. Generic filename, generic func name for "to_string"ing of a **very specific type**